cmd/uhc/whoami/cmd.go

```diff
@@ -91,23 +93,26 @@ func run(cmd *cobra.Command, argv []string) {
 		os.Exit(1)
 	}
 
-	request := connection.Get().Path("/api/accounts_mgmt/v1/current_account")
+	// Create and populate the request:
+	request := connection.AccountsMgmt().V1().CurrentAccount().Get()
+
+	// Buffer for pretty output
+	buf := new(bytes.Buffer)
 
 	// Send the request:
 	response, err := request.Send()
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "Can't send request: %v\n", err)
 		os.Exit(1)
 	}
-	status := response.Status()
-	body := response.Bytes()
-	if status < 400 {
-		err = dump.Pretty(os.Stdout, body)
-	} else {
-		err = dump.Pretty(os.Stderr, body)
-	}
```

Comment:
This will change the way that errors are reported. If the user doesn't have permission to get that resource, for example, we will still print the error response to `os.Stdout` instead of `os.Stderr`. Is that intended?

Reply:
Overlook on my part.